x/oracle/types/constants.go

```diff
@@ -0,0 +1,25 @@
+package types
+
+// nolint
+const (
+	DoNotModify = "[do-not-modify]"
+
+	MaxNameLength        = 128
```

Comment:
I think we can entirely remove this file. There is no need to check for maximum values on our chain since this is something related to Band and not Desmos.